lib/record_loader/flowcell_type_request_type_loader.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+# This file was automatically generated via `rails g record_loader`
+
+# RecordLoader handles automatic population and updating of database records
+# across different environments
+# @see https://rubydoc.info/github/sanger/record_loader/
+module RecordLoader
+  # Creates the specified plate types if they are not present
+  class FlowcellTypeRequestTypeLoader < ApplicationRecordLoader
```

Comment:
The comment says create specified plate type but looks like this is referring to flowcell types?

Reply:
I think this was something to do with the previous version of record_loader. The comment was autogenerated. I'll correct it, and all other places where it was autogenerated too.